### DISCUSSION TITLE
A view from an amateur developer

### DISCUSSION THREAD

**Discussion by wsw70** [tailwindlabs/tailwindcss#12125] [Kind words]
I've been developing for 20 years or so as an amateur, usually for my own needs, sometimes for work and also FOSS.

While I am mostly a backend guy, I like doing front code quite a lot. I have been using Vue with Quasar (a Vue framework with layout, components, classes, etc.).

Because everyone was talking about TailwindCSS I gave it a quick try once. I could not see the value, it was yet another set of keywords to learn and I had CSS for that. 

A year or so later (early 2023) I was still hearing about TailwindCSS and how great it was. I forced myself to use it when rewriting my home dashboard.

Initially it was hard, mostly because I had to switch back and forth between the docs and my code. But when I get used to it, oh boy, it was fast. 

Now I like it a lot and I try to make my frameworks work with it, replacing their own settings. **Preflight** is very useful there to start with a clean base.

Another **collateral** advantage is that I learned about CSS by reading the docs, as what TailwindCSS brings is usually a useful shortcut and uses useful CSS code. For someone who is really amateur in what I do it really helps.

**Comment by Matilda1185** [comment]
Thanks for sharing. I get a new idea from this. [MyEnvoyAir](https://www.myenvoyair.net/)
